alphacsc/tests/conftest.py

```diff
@@ -38,6 +35,18 @@ def X(rng, n_trials):
     return rng.randn(n_trials, N_CHANNELS, N_TIMES)
```

Comment:
Is it possible to have `n_trials=N_TRIALS` in the fixture, in order to avoid the extra parametrization?

Reply:
Unfortunately it does not work. I had tried it before.

But instead I created a fixture `def n_trials()` and it worked.

Reply:
Nice one!